deployment/src/main/java/io/quarkiverse/wiremock/devservice/WireMockServerConfig.java

```diff
@@ -40,4 +40,12 @@ public interface WireMockServerConfig {
      */
     @WithDefault("src/test/resources")
     String filesMapping();
+
+    /**
+     * If global response templating should be enabled for WireMock
+     *
+     * @see <a href="https://wiremock.org/2.x/docs/response-templating/">https://wiremock.org/2.x/docs/response-templating/</a>
```

Comment:
Let's link to the 3.X docu since the configuration options have changed: https://wiremock.org/3.x/docs/response-templating/

Reply:
Obsolete